main.js

```diff
@@ -44,15 +45,24 @@ const DEFAULT_EVENT_TEMPLATE_STRING = JSON.stringify(DEFAULT_EVENT_TEMPLATE)
 
 const run = async () => {
   try {
-    const relay = core.getInput('relay', { required: true })
+    const relayString = core.getInput('relay', { required: false })
```

Comment:
Could it be possible to mark the `relay` as deprecated using, for example, `core.warning`?

Reply:
Done in https://github.com/theborakompanioni/nostr-action/pull/166/commits/11b8193ffc0ea0b8d47126ecdead86691579c1cf.